Dockerfile.dev

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.17-alpine3.14 as builder
+FROM golang:1.18-alpine3.14 as builder
```

Comment:
```suggestion
FROM golang:1.18-alpine3.15 as builder
```

Reply:
Do we need to make changes to the prod Dockerfile?

Reply:
I'll change those as well!